notebooks/en/agent_data_analyst.ipynb

```diff
@@ -0,0 +1,1101 @@
+{
```

Comment:
<p>Below the code cell or here, I'd add a line or two about the HF Inference API here too. i.e. we are using it to run inference seamlessly without dealing with deployment worries and such.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/153/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>